freebsd-geom-exporter/src/main.rs

```diff
@@ -0,0 +1,129 @@
+// vim: tw=80
```

Comment:
comment not needed?

Reply:
By default vim sets it to 100 for .rs files, and I don't like that,